prow/config.yaml

```diff
@@ -15826,6 +15826,31 @@ periodics:
       secret:
         secretName: service-account
 
+# This is like the test-infra janitor but tailored for federation
+# projects
+- interval: 24h
+  agent: kubernetes
+  name: maintenance-fedtidy
+  spec:
+    containers:
+    - args:
+      - --project=k8s-jkns-e2e-gce-f8n-1-7
+      - --dns-zone=release1-7-test-f8n-k8s-io
+      command:
+      - /fedtidy
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      image: gcr.io/k8s-testimages/fedtidy:0.1
```

Comment:
can you commit the image to /images as well?

Reply:
I can for sure. Tricky part is getting the binary. What's the recommendation?

Reply:
^^